### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Mixin transformation of me.modmuss50.optifabric.mod.Text failed I am on 1.19.3

### DISCUSSION THREAD

**Discussion by xdminecrafter** [FabricMC/fabric-api#3646] [Player support]
### Minecraft Logs

```txt
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Mixin transformation of me.modmuss50.optifabric.mod.Text failed
```

### Description

![Capture](https://github.com/FabricMC/fabric/assets/163227292/f24ce6b9-d8d2-45a8-9e4f-d8d1f291b9a3)

**Comment by Poopooracoocoo** [comment]
You've got an outdated version of OptiFabric that was incompatible with Fabric Loader 0.15.0 and above.

Update it: https://www.curseforge.com/minecraft/mc-mods/optifabric/files/5025647

Also I advise against using OptiFabric and OptiFine. There are [mods you can replace OptiFine with](https://optifine.alternatives.lambdaurora.dev/) that'll run better and are more compatible with other mods.
